smoke-tests/src/weblogic/src/main/docker/wls/container-scripts/create-wls-domain.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
All these files in this directory, where are they from? Who is the original author and what is their license?

Reply:
I wrote them based on examples in https://github.com/oracle/docker-images/tree/master/OracleWebLogic/dockerfiles/12.2.1.4/container-scripts and other relevant subprojects.